gym_car_race/SelfDriveEnv.py

```diff
@@ -477,7 +478,8 @@ def _move_forward(self, dist):
         return (x + dist * cos(rad_angle), y + dist * sin(rad_angle))
         
     def set_pos(self, coords):
-        self.pos = coords
+        self.pos = [coords[0][0], coords[0][1]]
+        print("set", self.pos)
```

Comment:
Hi! I'm not so sure that the way that the `set_pos()` is used in line 220 would be compatible with the way you set `self.pos` because after `_calc_avg_pos()` is run, the `coords` passed into `set_pos()` is a list with 2 elements: x and y.

Reply:
Hi, i feel like you are looking at an old version. I change this part later. I am not very familiar with Git but if you can see the final version in "files changed" above.

Reply:
For this part, I found that if we draw a new map assigning only 1 start/final '_calc_avg_pos()' will not return a coordianate like [0, 1]. But something like [(0,1)]. So I changed a little bit in '_calc_avg_pos()', and revoke my change in "car.set_pos".

Reply:
Ah ok yeah that makes sense :) my bad.